### ISSUE TITLE
docs are broken/fail to build for 0.5.1

### ISSUE THREAD

**Issue by asutherland** [mozilla/rkv#81]
Going to the link from the README of https://docs.rs/rkv/ says "The requested resource does not exist."

Bringing up the crates page at https://docs.rs/crate/rkv/0.5.1 says they failed to build, linking to the log at https://docs.rs/crate/rkv/0.5.1/builds/117580

**Comment by mykmelez** [comment]
That log says the build failed while compiling ordered-float because "error[E0658]: the `#[repr(transparent)]` attribute is experimental (see issue #43036)":

```
   Compiling ordered-float v1.0.1
    …
error[E0658]: the `#[repr(transparent)]` attribute is experimental (see issue #43036)
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/ordered-float-1.0.1/src/lib.rs:44:1
   |
44 | #[repr(transparent)]
   | ^^^^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(repr_transparent)] to the crate attributes to enable
```

rkv switched back from new-ordered-float to ordered-float in #79. `#![feature(repr_transparent)]` was stabilized in https://github.com/rust-lang/rust/pull/51562. But that build appears to be using a version of rust from before that stabilization:

```
$ rustc --version
rustc 1.28.0-dev (b9bf4f162 2018-06-02)
```

I've added the comment https://github.com/onur/docs.rs/issues/23#issuecomment-426054041 to https://github.com/onur/docs.rs/issues/23, which tracks issues with building crate docs for docs.rs. But a browse of https://github.com/onur/docs.rs/issues shows unanswered related issues going back a month, such as https://github.com/onur/docs.rs/issues/239, https://github.com/onur/docs.rs/issues/235, https://github.com/onur/docs.rs/issues/233, and https://github.com/onur/docs.rs/issues/231. So this may not get fixed quickly.

In the meantime, a workaround is to generate and browse docs locally: `cargo doc --open`. Alternately, you can browse the 0.5.0 docs, as that version is identical, except for the change in the ordered-crate dependency: https://docs.rs/crate/rkv/0.5.0

Although this is ultimately a docs.rs problem, I'll leave this issue open until docs.rs is fixed to ensure we track and confirm the fix for rkv.

**Comment by mykmelez** [comment]
The docs.rs repository has moved from onur to rust-lang-nursery, so I updated all the links in the previous comment.

Also, according to https://github.com/rust-lang-nursery/docs.rs/issues/23#issuecomment-431617477, the rustc on docs.rs has been updated to a recent nightly, although failed builds haven't been queued to rebuild yet.

Finally, https://github.com/rust-lang-nursery/docs.rs/issues/246 notes that docs.rs appears to be down. So it isn't currently possible to employ the workaround of loading the 0.5.0 docs. You'll need to generate them locally.
